modules/network-observability-loki-install.adoc

```diff
@@ -34,7 +33,7 @@ There are several ways you can install Loki. One way is by installing the Loki O
 +
 . Create a `Secret` YAML file. You can create this secret in the web console or CLI. 
 
-.. Using the web console, navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. It is important to create this YAML file in the `netobserv` namespace that uses the `access_key_id` and `access_key_secret` to specify your credentials. 
+.. Using the web console, navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. Use the `access_key_id` and `access_key_secret` to specify your credentials. 
```

Comment:
I think that this information needed to be removed after [this](https://github.com/openshift/openshift-docs/pull/53263#discussion_r1054026176) discussion in the original PR where we discussed removing the step of using `oc create` to create the `netobserv` namespace.